pkg/lib/bundle/generate_test.go

```diff
@@ -303,3 +304,43 @@ func TestGenerateFunc(t *testing.T) {
 	require.NoError(t, err)
 	require.EqualValues(t, output, string(annotationsBlob))
 }
+
+func TestGenerateCSVlessBundle(t *testing.T) {
```

Comment:
I think this should be written as a table test, and should verify the contents of the output directory.

Reply:
Well, I did not do this here because I only want to see if they fail. The content comparison is done in `TestGenerateCSV` in synthesize/generate_test because they are the same test.

Reply:
Please let me know if you still think we should compare the content in here too. Thanks

Reply:
I think these tests change as a result of the decision to move the generation logic to a separate command, right?

Reply:
yes, it is gone now. I will push the changes.